src/corpus.jl

```diff
@@ -226,7 +257,25 @@ hash_function!(crps::Corpus, f::TextHashFunction) = (crps.h = f; nothing)
 # Standardize the documents in a Corpus to a common type
 #
 ##############################################################################
-
+"""
+    standardize!(crps, documentType)
+
+A Corpus may contain many different types of documents, it is generally more convenient
+to standardize all of the documents in a corpus using a single type.
+This can be done using the standardize! function.
```

Comment:
I feel that this can be summed up to a single sentence, something like `Standaradize all documents of 'crps' to 'documentType'`.